src/main/java/com/mailjet/client/transactional/TransactionalEmail.java

```diff
@@ -0,0 +1,192 @@
+     * Insert the file into the HTML code of the email by using cid:Filename, or, if you have specified ContentID, by using cid:ContentID.
+     * The total size of attachments(including inline) should not exceed 15 MB.
+     */
+    @Singular
+    private List<Attachment> inlinedAttachments;
+
+    /**
+     * Indicates the processing priority inside your account (API Key) scheduling queue.
+     * Equivalent of using X-Mailjet-Prio header through SMTP.
+     * Default value: 2
+     */
+    private Integer priority;
+
+    /**
+     * Specifies a custom campaign name, to which all messages with this property value will be assigned.
+     * If the campaign doesn't already exist it will be automatically created in the Mailjet system.
+     * API users are allowed to send multiple emails from the same campaign name to the same contacts.
+     * To block this feature, use the DeduplicateCampaign property.
+     * Equivalent of using X-Mailjet-Campaign header through SMTP.
+     */
+    private String customCampaign;
+
+    /**
+     * Enables or disables the option to send messages from the same campaign to the same contact multiple times.
+     * Equivalent of using X-Mailjet-DeduplicateCampaign header through SMTP.
+     * Default value: false
+     */
+    private Boolean deduplicateCampaign;
+
+    /**
+     *  Enable or disable open tracking on this message.
+     *  This property will overwrite the preferences selected in the Mailjet account.
+     *  Equivalent of using X-Mailjet-TrackOpen header through SMTP.
+     */
+    private TrackOpens trackOpens;
+
+    /**
+     *  Enable or disable click tracking on this message.
+     *  This property will overwrite the preferences selected in the Mailjet account.
+     *  Equivalent of using X-Mailjet-TrackClick header through SMTP.
+     */
+    private TrackClicks trackClicks;
+
+    /**
+     * Select a user-defined custom ID.
+     */
+    private String customID;
+
+    /**
+     *  Defines a payload attached to the message.
+     *  The Parse API will return the values in the payload.
+     *  Any standard format can be used - XML, JSON, CSV etc.
+     *  Equivalent of using X-MJ-EventPayload header through SMTP.
+     */
+    private String eventPayload;
+
+    /**
+     *  URL tags to append every URL link in the message.
+     *  The user needs to provide the query between the ? and # characters in the URL.
+     *  The URLTags value needs to be URL-encoded.
+     */
+    @SerializedName("URLTags")
+    private String urlTags;
+
+    /**
+     * Additional email headers.
+     */
+    @Singular
+    private Map<String, String> headers;
+
+    /**
+     * Adds variable used to modify the content of your email.
+     * Specified as {var_name}:{var_value} pairs.
+     * Enter the information in the template text / HTML part by using the [[var:{var_name}]] format.
+     * Equivalent of using X-MJ-Vars header through SMTP.
+     */
+    @Singular
+    private Map<String, String> variables;
```

Comment:
Maybe I am missing something, but what if a variable is an object or an array?

Reply:
yeah, the variable can be an array. Added possibility to pass an Object, that will be serialized as JSON